lisa/node.py

```diff
@@ -516,6 +528,16 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self.shell.initialize()
         self.os: OperatingSystem = OperatingSystem.create(self)
         self.capture_system_information("started")
+        if (
```

Comment:
The fix should be in the test case level, in case some mariner case needs to test `wheel` functionality. Please check with test case owner to fix it.